freqtrade/plot/plotting.py

```diff
@@ -451,13 +451,13 @@ def load_and_plot_trades(config: Dict[str, Any]):
     plot_elements = init_plotscript(config)
     trades = plot_elements['trades']
     pair_counter = 0
-    for pair, data in plot_elements["tickers"].items():
+    for pair, data in plot_elements["ohlcv"].items():
         pair_counter += 1
         logger.info("analyse pair %s", pair)
-        tickers = {}
-        tickers[pair] = data
+        ohlcv = {}
+        ohlcv[pair] = data
 
-        dataframe = strategy.analyze_ticker(tickers[pair], {'pair': pair})
+        dataframe = strategy.analyze_ticker(ohlcv[pair], {'pair': pair})
```

Comment:
While not directly related to these changes ... 
i think this can be simplified to `dataframe = strategy.analyze_ticker(data, {'pair': pair})`

the above dict (`ohlcv`) is not used again in this ... ?